### ISSUE TITLE
Enhance: Our Clients says section

### ISSUE THREAD

**Issue by Pragati-3003** [AbhiPatel10/AISKCON-CONSTRUCTION#247]
I would like to change the our clients says section by adding a slider with cool animation, box shadow and other effects. 
![IMG_20230528_185205](https://github.com/AbhiPatel10/AISKCON-CONSTRUCTION/assets/132778037/8ec580a1-e437-490d-90e7-5c3068aeea0c)
So, please assign this issue to me under GSSOC'23.

**Comment by Neilblaze** [comment]
@Pragati-3003 Do you mean implementing a carousel? Feel free to drop suggestions (such as low-level prototype designs) on how you're willing to bring changes to the current design. Once it gets approved, you'll be assigned & then can start working on it.

**Comment by RIYASANVI7** [comment]
i would like to bring modifications as the page looks simple by css .Kindly assign this project to me.

**Comment by Pragati-3003** [comment]
> @Pragati-3003 Do you mean implementing a carousel? Feel free to drop suggestions (such as low-level prototype designs) on how you're willing to bring changes to the current design. Once it gets approved, you'll be assigned & then can start working on it.

![IMG_20230529_132652](https://github.com/AbhiPatel10/AISKCON-CONSTRUCTION/assets/132778037/010b950d-f7de-4cd2-baea-b1d6909d4552)
I want to implement the above carousel, kindly inform me if you wanna any other changes.

**Comment by Lavesh12** [comment]
please assign me this issue

**Comment by Neilblaze** [comment]
@Pragati-3003 are you still on this?

**Comment by Pragati-3003** [comment]
> @Pragati-3003 are you still on this?

Yes, I would like to work on this , so please assign this to me.

**Comment by Neilblaze** [comment]
Go ahead! @Pragati-3003
